buildenv/jenkins/JenkinsfileBase

```diff
@@ -242,19 +249,23 @@ def setupParallelEnv() {
 // num cannot be greater than slave machines / 2
 def getNumMachines(){
 	int num = params.NUM_MACHINES ? params.NUM_MACHINES.toInteger() : 1
-	def slaves = nodesByLabel(LABEL).size()
-	if (slaves == 1) {
-		echo "Only 1 slave machine available. Set num to 1"
-		return slaves
-	}
-	int limit = slaves / 2
+	int limit = getHalfSlaves()
 	if (num > limit) {
 		echo "Number of slave machines is ${slaves}. Number of machines cannot be greater than ${slaves} / 2 (=${limit}). Set num to ${limit}"
```

Comment:
${slaves} is moved to getHalfSlaves(). Please update the msg.

Reply:
updated